library/Garp/Content/Manager.php

```diff
@@ -388,6 +388,11 @@ public function relate(array $options) {
         $bindingModel  = isset($bindingModel) ? 'Model_' . $bindingModel : null;
         $bidirectional = isset($bidirectional) ? $bidirectional : null;
 
+        if (isset($bindingModel)) {
+            $bindingModel = new $bindingModel();
+            $bindingModel->setCmsContext(true);
+        }
+
```

Comment:
Wanneer je nu de ContentManager gebruikt buiten de cms context dan krijg je nu problemen. Om te checken of we in de cms context zitten kan je het volgende doen.
```
if (isset($bindingModel) && $this->_model->isCmsContext()) {
    $bindingModel = new $bindingModel();
    $bindingModel->setCmsContext(true);
}
```
Maar durf nog niet helemaal te zeggen of dit nog gevolgen heeft voor de  Garp_Content_Relation_Manager.
@harmenjanssen Kan jij hier nog even naar kijken?